recipes/lxml/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "3.6.2" %}
```

Comment:
I need this specific version for pycsw. See https://github.com/geopython/pycsw/pull/456
We can update it later to the latest release on the feedstock.